### PR TITLE
Update pasta

### PR DESCRIPTION
**Pull request by sjanssen2**
Updating to the latest upstream version 1.9.0.
This PR should also avoid the upstream mechanism to source binary dependencies from a secondary github repo. Instead according bioconda dependencies are used.

Currently, the optional `--tree-estimator raxml` parameter leads to failing tests (only OSX); but I feel this is due to issues on upstream, not the recipe itself. Unfortunately, upstream is lacking public CI/CD, but I suggest to leave fixing this problem to upstream. Note that primary use of pasta is NOT affected.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by sjanssen2** [comment]
@BiocondaBot please add label

File: recipes/pasta/meta.yaml [hunk 1]
```diff
@@ -37,12 +56,22 @@ test:
...
-    - raxmlp -h | grep raxmlHPC
+    - raxml -h | grep -i raxml
+    - raxmlp -h | grep -i raxml
+    - export PASTA_LOGGING_LEVEL=DEBUG; export PASTA_LOGGING_FORMAT=RICH; run_pasta.py -i $PREFIX/share/pasta/data/small.fasta
+    # we need support from upstream to fix this special execution for OSX specific - export PASTA_LOGGING_LEVEL=DEBUG; export PASTA_LOGGING_FORMAT=RICH; run_pasta.py -i $PREFIX/share/pasta/data/small.fasta --tree-estimator raxml
 
 about:
   home: https://github.com/smirarab/pasta
-  license: GNU General Public License v3 or later (GPLv3+)
+  license: GPLv3
```
Comment (mencian) [inline recipes/pasta/meta.yaml:68]:
```suggestion
  license: GPL-3.0-or-later
```
Please use a SPDX identifier, i.e., `GPL-3.0-only` or `GPL-3.0-or-later`.

**Comment by mencian** [comment]
LGTM, thanks @sjanssen2!

**Comment by Sann5** [comment]
Hello @sjanssen2. Quick question why is hmmer pinned to 3.1b2? I just tried unpinning it in the sepp and pasta recipe ([in this PR](https://github.com/bioconda/bioconda-recipes/pull/48294)) and the build was successful.

**Comment by sjanssen2** [comment]
Hi @Sann5 I pinned this version, because later versions return a slightly different output format and I have not found the time to properly check if this might effect pasta / sepp. I suggest you test/compare a couple of inputs to see if results are identical

**Pull request merged**